src/api/users.rs

```diff
@@ -0,0 +1,105 @@
+//! The `users` API: `/user`, `/users`
+//!
+//! Endpoints under this deal with querying public, and private information
+//! about specific users.
+//!
+//! [Official Documentation][docs]
+//!
+//! [docs]: https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28
+
+use chrono::{DateTime, Utc};
+
+use crate::{models::gists::Gist, Octocrab, Page};
+
+/// Handler for GitHub's users API.
+///
+/// Created with [`Octocrab::users`]
+pub struct UsersHandler<'octo> {
+    crab: &'octo Octocrab,
+}
+
+impl<'octo> UsersHandler<'octo> {
+    pub(crate) fn new(crab: &'octo Octocrab) -> Self {
+        Self { crab }
+    }
+
+    /// List gists for the given username, allowing for pagination.
+    ///
+    /// See [GitHub API Documentation][docs] for details on `GET /users/{username}/gists`
+    ///
+    /// # Examples
+    ///
+    /// * Fetch 10 recent gists for the user with login "foouser":
+    /// ```no_run
+    /// # async fn run() -> octocrab::Result<()> {
+    ///     octocrab::instance()
+    ///         .users()
+    ///         .list_user_gists("foouser")
+    ///         .page(1u32)
+    ///         .per_page(10u8)
+    ///         .send()
+    ///         .await?;
+    /// # }
+    /// ```
+    ///
+    /// [docs]: https://docs.github.com/en/rest/gists/gists?apiVersion=2022-11-28#list-gists-for-a-user
+    pub fn list_user_gists(&self, username: impl AsRef<str>) -> ListGistsForUserBuilder<'octo> {
+        ListGistsForUserBuilder::new(self.crab, username.as_ref().to_string())
+    }
+}
```

Comment:
Official docs file this endpoint with the rest of gist endpoints. Would you prefer I move this handler into `api/gists.rs` as well?

Reply:
Sounds good to me.